test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -778,7 +779,8 @@ protected Settings featureFlagSettings() {
         for (Setting builtInFlag : FeatureFlagSettings.BUILT_IN_FEATURE_FLAGS) {
             featureSettings.put(builtInFlag.getKey(), builtInFlag.getDefaultRaw(Settings.EMPTY));
         }
-        featureSettings.put(FeatureFlags.TELEMETRY_SETTING.getKey(), true);
+        // Enabling Telemetry setting by default
+        featureSettings.put(TelemetrySettings.TRACER_ENABLED_SETTING.getKey(), true).put(FeatureFlags.TELEMETRY_SETTING.getKey(), true);
```

Comment:
Feature flags and node settings are separated.
```suggestion
        featureSettings.put(FeatureFlags.TELEMETRY_SETTING.getKey(), true);
```

Please move `put(TelemetrySettings.TRACER_ENABLED_SETTING.getKey(), true)` to `nodeSettings()`